src/get_estimation.py

```diff
@@ -4,327 +4,705 @@
 
 import time
 from .benchmark_mediation import *
-from rpy2.rinterface import RRuntimeError
+from rpy2.rinterface_lib.embedded import RRuntimeError
 import sys
 import pandas as pd
 import numpy as np
 
 
 def get_estimation(x, t, m, y, estimator, config):
-    if estimator == 'huber_IPW_R':
+    if estimator == "huber_IPW_R":
         x_r, t_r, m_r, y_r = [_convert_array_to_R(uu) for uu in (x, t, m, y)]
-        output_w = causalweight.medweight(y=y_r, d=t_r, m=m_r, x=x_r, trim=0.0,
-                                          ATET="FALSE", logit="TRUE", boot=2)
-        raw_res_R = np.array(output_w.rx2('results'))
+        output_w = causalweight.medweight(
+            y=y_r, d=t_r, m=m_r, x=x_r, trim=0.0, ATET="FALSE", logit="TRUE", boot=2
```

Comment:
```suggestion
            y=y_r, 
            d=t_r, 
            m=m_r, 
            x=x_r, 
            trim=0.0, 
            ATET="FALSE", 
            logit="TRUE", 
            boot=2
```

Reply:
Note that it is weird to use  `"FALSE"`, you probably want to use `False` instead. Same for  `True`

Reply:
I talked with judith about it. It seems to be R implementation, and shouldn't be modified